lib/project-data-updater.js

```diff
@@ -88,8 +86,7 @@ ProjectDataUpdater.prototype.checkForAndImportUpdates = function(info) {
         .then(done, done);
     });
   } else {
-    this.done(new Error(this.fullName + ': no ' +
-      module.exports.ABOUT_YML + ' updates imported'));
+    this.done(true);
```

Comment:
Yes, now I see what you mean. So what we should probably do is have a function signature of `done(err, noUpdate)` or something. I actually _detest_ boolean parameters, but in this case it's the most expedient solution (unless we do `this.noUpdate = true`, which even I will admit is a stretch).